### PR TITLE
User stat and update

### PR DESCRIPTION
**Pull request by sharminelow**

### CODE REVIEW

File: app_server/app/controllers/UserController.js [hunk 1]
```diff
@@ -134,6 +159,14 @@ var loginPayloadValidator = {
   }
 };
 
+var updateUserValidator = {
+  payload: {
+    alias: Joi.string().optional().default(null),
+    description: Joi.string().optional().default(null),
```
Comment (ndt93) [inline app_server/app/controllers/UserController.js]: `alias` shouldn't be optional or null. Otherwise we may not have anything to display about the user.

**Comment by ndt93** [comment]
In `Authenticator.js`, `Service.updateUserParticulars` that hasn't been renamed to `Service.updateUser`.

After refactoring a function's name, occurrences of the old name can be searched in all files in directories by e.g. `grep -rwn ./app ./test -e "updateUserParticulars"`. You text editor might have a feature for that also though.

**Comment by sharminelow** [comment]
Thanks for spotting the errors, big mistake on my part. Revised the code and you can take a look at it again. @ndt93

**Comment by ndt93** [comment]
Ok, good to merge. My authenticator tests last time should have covered that part :-P

**Pull request merged**